rickchurch/templates/base.html

```diff
@@ -2,16 +2,15 @@
 <html lang="en">
 
 <head>
-  {% block head %}
-  <title>rickchurch</title>
+  <title>{% block title %}{% endblock %}</title>
+  <meta charset="UTF-8"/>
```

Comment:
Closing this with `/>` is XHTML not HTML and it isn't consistent with our other meta tag(s)

Reply:
No idea what's XHTML, should I just do `<meta charset="UTF-8">` then?

Reply:
Yes